Header header
float64 speedfr
float64 speedfl
float64 speedrr
float64 speedrl
float64 speed_avr_r
float64 lat_acc_speed
uint8 parking_brake_on
uint8 air_cond_on
float64 steering_wheel_angle
uint16 steering_wheel_angular
float64 steering_wheel_tq
float64 pos_acc_pedal
uint8 brake_active_state
float64 brake_cylinder_press
float64 engine_speed
uint8 gear_trans_state
uint8 gear_seldisp
float64 pos_throttle
float64 cluster_odometer
float64 lon_acc_speed
float64 vehicle_speed_engine
float64 yaw_rate 